### DISCUSSION TITLE
Analytics Endpoints

### DISCUSSION THREAD

**Discussion by cguweb-com** [EcomPHP/tiktokshop-php#52] [Q&A]
Hello there,

Are there any plans to include the Analytics API, for endpoints like the following?
/analytics/202405/shop_products/performance
/analytics/202405/shop_products/{product_id}/performance

Chris

**Comment by nVuln** [comment]
already added it on `v2.6.0`

**Reply by cguweb-com** [reply]
Ahhh, yes!! Thank you!  I am having a version issue, however - please correct me if I'm wrong?  Continues to complain: "API version 202405 is the minimum requirement"

      $params = array(
        'start_date_ge'=>'2024-11-01',
        'end_date_lt'=>date('Y-m-d'),
      );
      $results = $client->Analytics->useVersion('202405')->getShopSkuPerformanceList($params);
